docs/source/gbd2017_models/causes/lower_respiratory_infections/index.rst

```diff
@@ -72,20 +72,13 @@ There are three sequelae associated with LRI. We are not tracking the long-term
 effects of Guillain-Barré syndrome (which can include paralysis, for example). However, since the prevalence of GBS is so low,
 there would probably not be much benefit in attempting to capture its long-term YLDs in addition to its short term YLDs.
 
-.. todo::
+.. note::
 
-   Describe more assumptions and limitations of the model.
+	Birth prevalence of LRI was allowed in the DISMOD modeling process for LRI. However, it was not reported as a final result in GBD 2017. LRI birth prevalence must therefore be retreieved using get_model_results('epi', 1258, age_group_id=164, measure_id=5) or get_draws('modelable_entity_id', 1258, source='epi', age_group_id=164, measure_id=5). 
 
-.. warning::
+.. todo::
 
-  Initial validation of the LRI cause model has indicated that prevalence (and 
-  therefore cause-specific mortality) of LRI is significantly underestimated for 
-  the early neonatal and late neonatal age groups (age group IDs 2 and 3). This 
-  may be because DISMOD models allow for birth prevalence of LRI, although birth 
-  prevalence is not a reported outcome of GBD. This issue should be investigated 
-  to see if we can incorporate birth prevalence into the LRI cause model to avoid 
-  underestimation of LRI in the neonatal age groups. Note: according to GBD 
-  modeler (Chris Troeger) LRI birth prevalence can be found using age_group_id 164.
+   Describe more assumptions and limitations of the model.
```

Comment:
age group id 164 refers to birth.

Reply:
maybe good to add a note saying that the remission is longer than 6 days (the prior bound on remission in the models is 7.3 days), so birth prevalence and 0-6 day prevalence should be nearly identical. 





12:05
Yep, the prior bound on remission in the models is 7.3 days.

Reply:
We don't see this in the data though I think because LRI has a high EMR... babies with LRI are dying in the first 6 days of life which causes LRI birth prevalence > LRI early neonatal prevalence

Reply:
oh awesome! would be nice to note.

Reply:
Agreed, that sounds like a good note to add.  Not sure where to put it -- maybe under Validation Criteria?